src/Casdoor.Client/CasdoorResourceClient.cs

```diff
@@ -0,0 +1,64 @@
+using System.Text;
+using System.Text.Json;
+using Casdoor.Client.Config;
+using Casdoor.Client.Entity;
+using Casdoor.Client.Exception;
+using Casdoor.Client.Util;
+
+namespace Casdoor.Client;
+
+public class CasdoorResourceClient
+{
+    private readonly CasdoorClientOptions _options;
+    private readonly CasdoorHttpClient _httpClient;
+
+    public CasdoorResourceClient(CasdoorClientOptions? options)
+    {
+        _options = options ?? throw new ArgumentNullException(nameof(options));
+        _httpClient = new CasdoorHttpClient(_options);
+    }
+
+    // CasdoorUserClient.cs
+    // TODO: what are `createdTime` and `description` for
+    public async Task<CasdoorUserResource> UploadResourceAsync(
+        string user, string tag, string parent, string fullFilePath,
+        Stream fileStream,
+        string createdTime = "", string description = "")
+    {
+        var queryMap = new Dictionary<string, string>
+        {
+            {"owner", _options.OrganizationName},
+            {"user", user},
+            {"application", _options.ApplicationName},
+            {"tag", tag},
+            {"parent", parent},
+            {"fullFilePath", fullFilePath},
+        };
+
+        using MemoryStream ms = new();
```

Comment:
It is also wrong, you need the [StreanContext](https://docs.microsoft.com/zh-cn/dotnet/api/system.net.http.streamcontent?view=net-6.0).
`Steam` is used to buffered reads or writes and `MemoryStream` is rarely used.